spec/presenters/redirect_versions_presenter_spec.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.describe RedirectVersionsPresenter do
+  describe 'last updated by' do
+    context 'user exists' do
+      it "presents the identifier before the @-sign in user's email" do
+        user = create(:user, email: 'example@test.com')
+        redirect_version = double('redirect_version', user: user)
+
+        expect(described_class.new(redirect_version).last_updated_by).to eq('example')
+      end
+    end
+
+    context 'user does not exist' do
+      it 'shows an helpful message' do
+        redirect_version = double('redirect_version', user: nil)
+
+        expect(described_class.new(redirect_version).last_updated_by).to eq('deleted user')
```

Comment:
I thought about making `described_class.new(redirect_version)` the subject but, each context has varying requirements for the `redirect_version` stub.